### ISSUE TITLE
Consume 100% del CPU

### ISSUE THREAD

**Issue by R7Marl** [microsoft/vscode#215616]
Type: <b>Performance Issue</b>

En los ultimos dias Visual Studio Code me consume todo el procesador cuando lo uso, esto pasó de un dia para el otro.

VS Code version: Code 1.90.1 (611f9bfce64f25108829dd295f54a6894e87339d, 2024-06-11T21:02:43.666Z)
OS version: Linux x64 6.8.0-35-generic snap
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 3 2200G with Radeon Vega Graphics (4 x 3698)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|4, 3, 2|
|Memory (System)|15.54GB (11.77GB free)|
|Process Argv|--force-user-env --no-sandbox --use-gl=angle --use-angle=swiftshader --crash-reporter-id 9f668921-54d7-4b4c-9f91-b18832ecb7c1|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|ubuntu|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|ubuntu|
|XDG_SESSION_TYPE|x11|
</details><details>
<summary>Process Info</summary>

```
CPU %	Mem MB	   PID	Process
    0	   159	  3915	code main
    0	    32	  3917	   zygote
    0	    32	  3918	   zygote
    0	   334	  3989	window [1] (● addGameController.js - queue-clash-lol-main - Visual Studio Code)
    0	    95	  5038	window [2] (Issue Reporter)
    0	    64	  3962	   utility-network-service
    0	   111	  4221	shared-process
    0	     0	  5051	     /bin/sh -c /usr/bin/ps -ax -o pid=,ppid=,pcpu=,pmem=,command=
    0	     0	  5052	       /usr/bin/ps -ax -o pid=,ppid=,pcpu=,pmem=,command=
    0	    80	  4222	fileWatcher [1]
    0	   127	  4367	extensionHost [1]
    0	   111	  4384	     electron-nodejs (tsserver.js )
    0	   191	  4385	     electron-nodejs (tsserver.js )
    0	    80	  4413	       electron-nodejs (typingsInstaller.js typesMap.js )
    0	    64	  4424	     /snap/code/161/usr/share/code/code /snap/code/161/usr/share/code/resources/app/extensions/json-language-features/server/dist/node/jsonServerMain --node-ipc --clientProcessId=4367
```

</details>
<details>
<summary>Workspace Info</summary>

```
|  Window (● addGameController.js - queue-clash-lol-main - Visual Studio Code)
|    Folder (queue-clash-lol-main): 73 files
|      File types: js(35) json(2) env(1) example(1) gitignore(1) md(1)
|      Conf files: package.json(1);
```

</details>
<details><summary>Extensions (1)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-language-pack-es|MS-|1.89.2024050109

</details>

**Comment by shelvesdragon** [comment]
Unfortunately, I don't speak Spanish, but based on machine translation and your provided info, you seem to be experiencing a lot of CPU usage on the Ubuntu snap version of VSCode. Note that snap provides automatic updates by design, unless you tell it to keep a specific version. The high resource usage is likely caused by a temporary fix in version 1.90.1 that switched the OpenGL backend to software emulation in order to prevent the crashes reported with version 1.90.0 in #212494 .
If it is related, your options for now are mentioned in this comment: https://github.com/microsoft/vscode/issues/212494#issuecomment-2168072242
Hope this helps!

**Comment by deepak1556** [comment]
Merging to https://github.com/microsoft/vscode/issues/212494, please follow the issue for updates.
